### ISSUE TITLE
Loss does not decrease?

### ISSUE THREAD

**Issue by tangbohu** [fabiopoiesi/dip#4]
In my experiments, the loss remains to be around 0.8-1.1 even after 300k iterations.  Is it right?

**Comment by fabiopoiesi** [comment]
Yes, it is possible. Do you have some tensorboard plots?
If you are interested, we are now using different training parameters to learn faster.

**Comment by tangbohu** [comment]
Thanks @fabiopoiesi ! I am very interested in this work. I have visualized the tensorboard, it seems OK. But if you have different training parameters, would you mind sharing with us? Besides, could you share us the test codes to get the results reported in table 1 and 2 of the paper. 

Thanks again for your kind reply!

**Comment by fabiopoiesi** [comment]
Some tips:
- do not train on multiple gpus (for some reason the performance decreases), so avoid using nn.DataParallel
- train for 10 epochs using: 
```
optimizer = optim.SGD(net.parameters(), lr=1e-1, momentum=0.9, weight_decay=5e-5, nesterov=False)
scheduler = optim.lr_scheduler.StepLR(optimizer, step_size=3, gamma=0.1)
```
you might achieve the same performance with this parameters, but a little quicker.

For the second request, I'll get back to you soon.

**Comment by tangbohu** [comment]
Thanks! I am looking forward to seeing the evalution codes.

**Comment by tangbohu** [comment]
Dear @fabiopoiesi , I wonder whether the test codes are available now?

**Comment by fabiopoiesi** [comment]
I haven't had time yet. 
But the code for Tab. 1 is already there, have you checked the file **benchmark_3dmatch_pre.py**?
For Tab. 2, I have to prepare it.

**Comment by tangbohu** [comment]
Thanks. I will try it first.

**Comment by fabiopoiesi** [comment]
Have you had chance to try it?

**Comment by wenli4313** [comment]
> In my experiments, the loss remains to be around 0.8-1.1 even after 300k iterations. Is it right?

Hello，How do you get the final_chkpt.pth?Thank you very much.
